features/cellular/framework/AT/AT_CellularDevice.cpp

```diff
@@ -365,5 +365,15 @@ void AT_CellularDevice::modem_debug_on(bool on)
 
 nsapi_error_t AT_CellularDevice::init_module()
 {
+    CellularInformation *information = open_information();
```

Comment:
only for printing so this code block could be inside "#if MBED_CONF_MBED_TRACE_ENABLE